CONTRIBUTING.md

```diff
@@ -127,7 +127,12 @@ This is a rough outline of what a contributor's workflow looks like:
 - Run tests and make sure all tests pass.
 - Push your changes to a branch in your fork of the repository and submit a pull request.
 - Your PR will be reviewed by two maintainers, who may request some changes.
-- Our CI systems automatically test all the pull requests.
+  * Once you've made changes, your PR must be re-reviewed and approved.
+  * If the PR becomes out of date, you can use GitHub's 'update branch' button.
+  * If there are conflicts, you can rebase (or merge) and resolve them locally. Then force push to your PR branch.
+    You do not need to get re-review just for resolving conflicts, but you should request re-review if there are significant changes.
+- Our CI system automatically tests all pull requests.
+- Our bot will merge your PR. It can be summoned by commenting `/merge` or adding the `S: CanMerge` label (requires tests to pass and two approvals, you might have to ask your reviewer to do this).
```

Comment:
```suggestion
- Our bot will merge your PR. It can be summoned by commenting `/merge` or adding the `S: CanMerge` label (requires tests to pass and two approvals. You might have to ask your reviewer to do this).
```